### PR TITLE
Skip unreliable time measurement assertions on Windows.

### PR DESCRIPTION
**Pull request by ChrisHines**

### CODE REVIEW

File: scheduler/stack_test.go [hunk 1]
```diff
@@ -72,8 +73,11 @@ func TestServiceStack_Select_Size(t *testing.T) {
 		t.Fatalf("bad: %#v", size)
 	}
 
+	// Note: On Windows time.Now currently has a best case granularity of 1ms.
+	// We skip the following assertion on Windows because this test usually
+	// runs to fast to measure an allocation time on Windows.
```
Comment (dadgar) [inline scheduler/stack_test.go]: Small nitpick but `runs to fast` -> `runs too fast`

**Comment by dadgar** [comment]
LGTM

File: scheduler/stack_test.go (hunk 1 above)
Comment (ChrisHines) [inline scheduler/stack_test.go]: Good catch. I will fix and update the PR when I get a chance.

**Comment by ChrisHines** [comment]
@dadgar PTAL

**Comment by cbednarski** [comment]
Thanks @ChrisHines!

**Pull request merged**